docs/changelog.rst

```diff
@@ -85,6 +85,10 @@ See :ref:`upgrade-guide-2.6` for help upgrading to the new version.
     ``CMAKE_INTERPROCEDURAL_OPTIMIZATION``, ``set(CMAKE_CXX_VISIBILITY_PRESET
     hidden)``.
 
+  * Helper functions ``pybind11_strip``, ``pybind11_extension``,
+    ``pybind11_find_import`` added (see ``tools/pybind11Config.cmake.in`` for
+    docs).
```

Comment:
Looks fantastic, but as the CMake integration gets fancier, can we have coverage in RTD? You could just copy the text from `tools/pybind11Config.cmake.in` and expand it very slightly. As is, it  is not going to be very easy to find for users.

Reply:
Docs pull the page in from the CMake files now, and reformatted this to look more like FindPython's docs. CMake's docs are built by Sphinx, by the way.